packages/cli/src/Interfaces.ts

```diff
@@ -890,3 +890,5 @@ export interface N8nApp {
 	externalHooks: IExternalHooksClass;
 	activeWorkflowRunner: ActiveWorkflowRunner;
 }
+
+export type userSettings = Pick<User, 'id' | 'settings'>;
```

Comment:
```suggestion
export type UserSettings = Pick<User, 'id' | 'settings'>;
```

Reply:
If you can also please remove the file-level exemption which caused this and have it affect only `IDatabaseCollections`.